registry.k8s.io/images/k8s-staging-sig-storage/images.yaml

```diff
@@ -200,6 +202,10 @@
     "sha256:1d9f524c0fd5d714cf274a0ddf5dceebeb1848349de5687e530c33dd2675e4ba": [ "v4.2.0" ]
     "sha256:2103ed076e38b87b877a3a58377fe546b5b6a9b83aff75bc9d79aa7a8f0569f2": [ "v4.3.0" ]
     "sha256:971cb526bc0108cca7d8efda32102629cb362f99bbd4879fd381bea1b2014ab4": [ "v4.4.0" ]
+- name: objectstorage-controller
```

Comment:
do you know why these got picked up?

Reply:
Yes. @BlaineEXE is trying to get COSI images uploaded to the staging repo.
https://github.com/kubernetes/k8s.io/pull/5846